internal/config/cgmgr/stats_test.go

```diff
@@ -1,91 +0,0 @@
-package cgmgr_test
```

Comment:
This file just tests UpdateWithMemoryStatsFromFile function which is deleted and no longer used